src/Microsoft.Fx.Portability/InternalServerErrorException.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Fx.Portability.Resources;
+using System;
+
+namespace Microsoft.Fx.Portability
+{
+    public class InternalServerErrorException : PortabilityAnalyzerException
+    {
+        private InternalServerErrorException(string message, string exception)
+            : base(string.Format(LocalizedStrings.InternalServerErrorMessage, message, exception))
+        { }
+
+        /// <summary>
+        /// This returns a properly formatted exception received from the server.
+        /// </summary>
+        public static InternalServerErrorException Create(string message, string exception)
+        {
+            var split = exception.Split(new[] { "\\r\\n", "\\r", "\\n" }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
You probably only need \\r and \\n, since \\r\\n will occur with that as well. Interesting thing I found: if you pass null in, it will split at all white space (looks like you only want to split at new lines, though); see the remarks at: https://msdn.microsoft.com/en-us/library/b873y76a.aspx